main.go

```diff
@@ -64,7 +64,8 @@ func main() {
 	if err != nil {
 		log.Fatalf("Expected comma-separated list of IPs: %v", err)
 	}
-	netQueue = NewNetfilterQueue(nfQueueNumber, ips)
+	netQueue = NewNetQueue(nfQueueNumber, ips)
```

Comment:
Yes, but in testing each test uses a different queue to avoid one test affecting other.
Also in the same host you could start multiple haproxies each of them using a different queue, so it can be configured.